SETUP.md

```diff
@@ -169,19 +169,23 @@ This option utilizes an installation script to do the setup, and it requires add
 >        ```{shell}
 >        databricks clusters start --cluster-id <CLUSTER_ID>`
 >        ```
-> * The script also requires the `zip` command line utility, which may not be installed. You can install it with:
->     ```{shell}
->     sudo apt-get update
->     sudo apt-get install zip
->     ```
+
 
 Once you have confirmed the databricks cluster is *RUNNING*, install the modules within this repository with the following commands:
 
 ```{shell}
 cd Recommenders
-./scripts/databricks_install.sh <CLUSTER_ID>
+./scripts/databricks_install.py <CLUSTER_ID>
 ```
 
+**Note** If you are planning on running through the sample code for operationalization [here](notebooks//05_operationalize/als_movie_o16n.ipynb), you need to prepare the cluster for operationalization. You can do so by adding an additional option to the script run:
+
+```{shell}
+./scripts/databricks_install.py --prepare-o16n <CLUSTER_ID>
```

Comment:
Would be good to explain where to get the <CLUSTER_ID>.

Reply:
+1

Reply:
added a note. It's described in an earlier section, but I added a note here as well.